### ISSUE TITLE
Application allow/deny list

### ISSUE THREAD

**Issue by scottmconway** [scottmconway/notifikator#1]
Notifikator should present the user with the option of specifying an allow or deny list. [AFWall](https://github.com/ukanth/afwall) has a particularly good UX for this.

Having a similar app list with icon/name and a checkbox per-app would be the best solution, but an editable text box where the user can specify package names will be the MVP for this issue (even though it's awful UX).

Taken from [this issue](https://github.com/koro666/notifikator/issues/3)

**Comment by neo-neo1** [comment]
Yes please!

**Comment by neo-neo1** [comment]
Yes please! 

Has this been implemented? Any compiled release? Thank you greatly. Been waiting a long time for this

**Comment by scottmconway** [comment]
Nope! I looked into it when I opened the issue, and realized that I really have no idea what I'm doing when it comes to UI design. For now I use a hard-coded list for my own deployment.

To get an achievable MVP for me, perhaps I could reduce it to having the app source a defined file or two in the app's external storage (allowlist.txt and/or blocklist.txt) that would contain one package name per line. On app init, it'd read the files and use the list as-is.

It'd be very simple to write, but annoying for a user to utilize.
